spec/models/manageiq/providers/redhat/infra_manager/refresh/refresher_4_async_spec.rb

```diff
@@ -33,6 +33,11 @@ def load_response_mock_for(filename)
 
   it "will perform a full refresh on v4.1" do
     EmsRefresh.refresh(@ems)
+
```

Comment:
you have a blank line here but in the other file you do not. same for line 40. I think its nice to have the blank line, but it should be the same in both files.

Reply:
done